pkg/console/install_panels.go

```diff
@@ -802,41 +817,12 @@ func addNetworkPanel(c *Console) error {
 			return err
 		}
 		mgmtNetwork.Method = selected
-
 		if selected == config.NetworkMethodStatic {
 			return showNext(c, dnsServersPanel, gatewayPanel, addressPanel)
 		}
 
 		c.CloseElements(dnsServersPanel, gatewayPanel, addressPanel)
-		if err := networkValidatorV.Show(); err != nil {
-			return err
-		}
-		spinner := NewFocusSpinner(g, networkValidatorPanel, fmt.Sprintf("Requesting IP through DHCP..."))
-		spinner.Start()
```

Comment:
It seems like we remove the spinner showing DHCP request status. Is this a UX regression?

Reply:
The spinner waiting for DHCP request is merging into "Applying Network Configurations". Shall we display the DHCP waiting status separately?

Reply:
I see. That works